sources/launcher/Stride.Launcher/Resources/Strings.Designer.cs

```diff
@@ -19,7 +19,7 @@ namespace Stride.LauncherApp.Resources {
     // class via a tool like ResGen or Visual Studio.
     // To add or remove a member, edit your .ResX file then rerun ResGen
     // with the /str option, or rebuild your VS project.
-    [global::System.CodeDom.Compiler.GeneratedCodeAttribute("System.Resources.Tools.StronglyTypedResourceBuilder", "16.0.0.0")]
+    [global::System.CodeDom.Compiler.GeneratedCodeAttribute("System.Resources.Tools.StronglyTypedResourceBuilder", "17.0.0.0")]
```

Comment:
Note that many of these changes were auto-generated in parallel with my change.  My changes were to rename the variables and strings to use 'Extension' rather than 'Plugin' to reduce confusion.  See related .resx file changes below.

Perhaps this file shouldn't be included in source control at all?

Reply:
Yup, a bunch of other files shouldn't be tracked, we'll clean this one with the rest afterward. Let's leave this one in for now.